modules/apps/map/map-common/test/liferay/MapBase.js

```diff
@@ -289,6 +279,8 @@ describe('MapBase', () => {
 		it('does nothing if there is no MapBase.GeocoderImpl', () => {
 			MapImpl.GeocoderImpl = null;
 
+			mapImpl = new MapImpl();
```

Comment:
Here, the `mapImpl` isn't being updated with the `MapImpl.GeocoderImpl = null;` line but its being tested in the line under it, so I made a new instance here so that the `GeocoderImpl` is set to `null`.